tests/test_fts5_search.py

```
@@ -0,0 +1,356 @@
+                    category="health"
+                )
+            ],
+            main_themes=["nutrition", "vitamins"],
+            executive_summary="Document about vitamin benefits and nutrition"
+        )
+
+        # Use unique pepper_id to avoid UniqueConstraint collision
+        with patch('local_insight_engine.persistence.repository.PersistentQASession') as MockSession:
+            def create_session_with_unique_pepper(**kwargs):
+                kwargs['pepper_id'] = f"test_pepper_nutrition_{uuid.uuid4().hex[:8]}"
+                return PersistentQASession(**kwargs)
```

Ruff (0.12.2)

Undefined name `PersistentQASession`

(F821)